README.md

```diff
@@ -98,10 +98,10 @@ For more details about the output files and reports, please refer to the
 
 ## Credits
 
-nf-core/riboseq was originally written by [Jonathan Manning](https://github.com/pinin4fjords) (Bioinformatics Engineer as Seqera) with support from [Altos labs](https://www.altoslabs.com/) and in discussion with [Felix Krueger](https://github.com/FelixKrueger) and [Christel Krueger](https://github.com/ChristelKrueger). We thank the following people for their input:
+nf-core/riboseq was originally written by [Jonathan Manning](https://github.com/pinin4fjords) (Bioinformatics Engineer at Seqera) with support from [Altos Labs](https://www.altoslabs.com/) and in discussion with [Felix Krueger](https://github.com/FelixKrueger) and [Christel Krueger](https://github.com/ChristelKrueger). We thank the following people for their input:
 
 - Anne Bresciani (ZS)
-- [Felibe Almeida](https://github.com/fmalmeida) (ZS)
+- [Felipe Almeida](https://github.com/fmalmeida) (ZS)
 - [Mikhail Osipovitch](https://github.com/mosi223) (ZS)
 - [Edward Wallace](https://github.com/ewallace) (University of Edinburgh)
 - [Jack Tierney](https://github.com/JackCurragh) (University College Cork)
```

Comment:
```suggestion
- [Jack Tierney](https://github.com/JackCurragh) (University College Cork)
- [Maxime U Garcia](https://github.com/maxulysse) (Seqera)
```

Reply:
Wait, I thought I'd got these...

Reply:
Ahh, this was merged directly to dev (oops), but don't worry, it's sorted